wordle-web/components/keyboard.vue

```diff
@@ -35,26 +35,23 @@
 import { Component, Vue, Prop } from 'vue-property-decorator'
 import { Letter, LetterStatus } from '~/scripts/letter'
 import { WordleGame } from '~/scripts/wordleGame'
+import { WordsService } from '~/scripts/wordsService'
```

Comment:
This is declared but never used, this is something your linter would catch if you run the command `npm run lintfix`
![image](https://user-images.githubusercontent.com/43832670/167057709-efd58c63-974a-4cff-a60a-aeeadd9b5433.png)